pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1331,6 +1337,12 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
           break;
         }
 
+        // If it's not a privileged installation preset guestAgent
+        // port binding address to localhost only.
+        if (!this.privilegedInstall) {
```

Comment:
> Should this only happen if not `config.kubernetes.ingress.localhostOnly`?

Yes, it would be good to add that check instead of always writing. 

> Should there be some sort of warning to the user?

I don't know if we currently have a way of informing users other than adding some documentation around this behavior. 🤔

Reply:
> > Should there be some sort of warning to the user?
> 
> I don't know if we currently have a way of informing users other than adding some documentation around this behavior.

We talked about this earlier today. Ideally the validator should be able to do this, but it doesn't have the information needed. I was thinking about setting `cfg.application.adminAccess = this.privilegedInstall` (but still not allow changing the setting via the API on Windows).

It kind of makes sense to me, as a privileged install on Windows is equivalent to granting admin access on macOS.

But I don't want to make the change anymore for the current release. @Nino-K is going to create a Github issue about it.